src/pools/fixedRate/FixedRateExchange.ts

```diff
@@ -366,10 +366,10 @@ export class FixedRateExchange {
     const estGas = await this.estSetRate(
       address,
       exchangeId,
-      await this.unitsToAmount(exchange.baseToken, String(newRate))
+      await this.amountToUnits(exchange.baseToken, String(newRate))
```

Comment:
why noy toString()?

Reply:
this is what was used above so I used to be consistent

Reply:
strange, we always use toString, don't know how that stayed there.

Reply:
I will change it

Reply:
feel free to update all String(value) in other places if you find more of this 😃

Reply:
ok, I will